mod/admin.php

```diff
@@ -209,9 +222,126 @@ function admin_content(&$a) {
 }
 
 /**
- * Admin Inspect Queue Page
+ * This function generates the "Federation Statistics" subpage for the admin
+ * panel. The page lists some numbers to the part of "The Federation" known to
+ * the node. This data includes the different connected networks (e.g.
+ * Diaspora, Hubzilla, GNU Social) and the used versions in the different
+ * networks.
+ *
+ * The returned string contains the HTML code of the subpage for display.
+ *
+ * @brief subpage with some stats about "the federation" network
```

Comment:
`@brief` should be at top and the detailed explanation should be below  `@brief`